packages/fxa-graphql-api/src/lib/auth.ts

```diff
@@ -13,10 +13,9 @@ export class SessionTokenAuth {
   @Inject(fxAccountClientToken)
   private authClient!: AuthClient;
 
-  public async lookupUserId(sessionToken: string): Promise<string> {
+  public async lookupUserId(sessionToken: string) {
```

Comment:
It is a little funky to keep the name `lookupUserId` since the return value is no longer a `uid`. Why not just `getSessionStatus`?

Reply:
true, i totally glossed over the fn name